jabgui/src/main/java/org/jabref/gui/walkthrough/Walkthrough.java

```diff
@@ -0,0 +1,323 @@
+                                                 .skipButton(Localization.lang("Skip to finish"))
+                                                 .build())
+                .build();
+        WalkthroughActionsConfig actions = WalkthroughActionsConfig.all(
+                Localization.lang("Continue"),
+                Localization.lang("Skip for Now"),
+                Localization.lang("Back"));
+        FullScreenStep paperDirectoryNode = WalkthroughNode
+                .fullScreen(Localization.lang("Configure paper directory"))
+                .content(
+                        new TextBlock(Localization.lang("Set up your main file directory where JabRef will look for and store your PDF files and other associated documents.")),
+                        new InfoBlock(Localization.lang("This directory helps JabRef organize your paper files. You can change this later in Preferences.")),
+                        new ArbitraryJFXBlock(_ -> new PaperDirectoryChooser())
+                )
+                .actions(actions)
+                .nextStepAction(walkthrough -> {
+                    if (frame.getCurrentLibraryTab() != null) {
+                        walkthrough.nextStep();
+                        return;
+                    }
+
+                    DialogService dialogService = Injector.instantiateModelOrService(DialogService.class);
+                    AiService aiService = Injector.instantiateModelOrService(AiService.class);
+                    GuiPreferences guiPreferences = Injector.instantiateModelOrService(GuiPreferences.class);
+                    StateManager stateManager = Injector.instantiateModelOrService(StateManager.class);
+                    FileUpdateMonitor fileUpdateMonitor = Injector.instantiateModelOrService(FileUpdateMonitor.class);
+                    BibEntryTypesManager entryTypesManager = Injector.instantiateModelOrService(BibEntryTypesManager.class);
+                    CountingUndoManager undoManager = Injector.instantiateModelOrService(CountingUndoManager.class);
+                    ClipBoardManager clipBoardManager = Injector.instantiateModelOrService(ClipBoardManager.class);
+                    TaskExecutor taskExecutor = Injector.instantiateModelOrService(TaskExecutor.class);
+
+                    LibraryTab libraryTab = LibraryTab.createLibraryTab(
+                            this.database,
+                            frame,
+                            dialogService,
+                            aiService,
+                            guiPreferences,
+                            stateManager,
+                            fileUpdateMonitor,
+                            entryTypesManager,
+                            undoManager,
+                            clipBoardManager,
+                            taskExecutor);
+                    frame.addTab(libraryTab, true);
+                    walkthrough.nextStep();
+                })
```

Comment:
Mh... i think there maybe a misunderstanding. I did not expect the walkthrough to add new logic to jabref, new dialogs etc. To me, a walkthrough like in vscode is just highlighting buttons, menus clickable things and describing for the user what happens.
This here looks like a first-time-setup-wizard

Reply:
Was clarified in call